make.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+
```

Comment:
It might be easier and shorter to use the npm workspaces by using this:

`npm run build --workspaces --if-present`

Reply:
The thing is: there's a definite order in which the packages should execute their `npm run build` when making from scratch, i.e. with a newly-cloned or `npm run clean`ed repository. The `make.sh` takes care of that order, next to generating some code.

Reply:
Depends...I'd say... I had it like that, but then again, I try to do not to too much on each script, so that I can relatively quickly run an `npm run build` to sync up if I made cross-package changes. In the end, `make.sh` does _everything_.

Reply:
Right! Therefore I usually have a build and a rebuild, where the latter does everything.

Reply:
So, you'd have a `build` that just compiles/transpiles/builds, and a `rebuild` that rebuilds everything (essentially from scratch)?